### ISSUE TITLE
Selecting file after successful stream crashes 

### ISSUE THREAD

**Issue by OverkillGuy** [keredson/gnomecast#40]
Hi Mr owner, love your work, was able to use Chromecast natively from Linux for first time in a while!

Today's (second) error is triggered by selecting a  file after having successfully streamed through an entire file. In my case I'm looking at a TV series, so I waited till end of one full episode and selected the second one.

I get the following:
```
Open clicked
File selected: /home/jiby/Downloads/Monkey Dust/Monkey Dust s01/Monkey Dust s01e03 (DivX 640x352).avi
WARNING:pychromecast.controllers:STOP command requested but no session is active.
WARNING:pychromecast.controllers:STOP command requested but no session is active.
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.4/threading.py", line 868, in run
    self._target(*self._args, **self._kwargs)
  File "/home/jiby/Documents/dev/foss/gnomecast/gnomecast.py", line 627, in update_transcoder
    self.transcoder.destroy()
  File "/home/jiby/Documents/dev/foss/gnomecast/gnomecast.py", line 160, in destroy
    self.p.terminate()
  File "/usr/lib/python3.4/subprocess.py", line 1686, in terminate
    self.send_signal(signal.SIGTERM)
  File "/usr/lib/python3.4/subprocess.py", line 1681, in send_signal
    os.kill(self.pid, sig)
```
Reliably reproduced after watching another episode.

Seems we're closing a session too early somewhere because streaming succeeded, despite the application not being done running.

I'll try to trace the issue more and submit another PR.

**Comment by OverkillGuy** [comment]
Seems to come from `Transcoder.destroy()` trying to destroy something that doesn't exist anymore (probably since the file has been transcoded, since the playback was complete?). The warning from pychromecast makes it look like the whole destroy method shouldn't even do anything if the object was already cleaned up beforehand.

**Comment by keredson** [comment]
thanks!

weird.  i've not seen errors when it tries to sigterm a process that's already finished.  i just added a check to the status to not send if it's already done.  i'll deploy, please let me know if this fixes it.

**Comment by keredson** [comment]
released 0.2.16

**Comment by OverkillGuy** [comment]
Confirm: can now select new file after a successful stream and there's no crash.
Thanks for the quick reaction time!
